### PR TITLE
update the translation of `Object()`

### PR DESCRIPTION
**Pull request by robert-jx**
### Description

update the translation of `Object()`
### Motivation

update the translation of `Object()`
### Additional details

none

### CODE REVIEW

**Comment by robert-jx** [comment]
@JasonLamv-t

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions.

File: files/zh-cn/web/javascript/reference/global_objects/object/object/index.md [hunk 1]
```diff
@@ -5,41 +5,47 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/Object
...
 
+### 返回值
+
+当调用或者构造 `Object()` 构造函数本身时，其返回值是一个对象：
+
+- 如果该值是 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null) 或者 {{jsxref("undefined")}}，它会生成并返回一个空对象。
+- 如果该值已经是一个对象，则返回该值。
+- 否则，它将返回与给定值对应的类型的对象。例如，传递 {{jsxref("BigInt")}} 基本类型会返回一个 `BigInt` 封装对象。
+
+当通过[扩展 `Object`](/zh-CN/docs/Web/JavaScript/Reference/Classes/extends#扩展对象) 的类的构造函数中的 [`super()`](/zh-CN/docs/Web/JavaScript/Reference/Operators/super) 隐式调用 `Object()` 时，它以 `new.target.prototype` 为原型初始化一个新对象。传递给 `super()` 的任意值都将被忽略——例如，即使你传递一个数字，构造函数中的 `this` 值也不会成为 {{jsxref("Number")}} 实例。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/object/index.md:32]: 在 Java 或 C++ 里面，我们称其为“继承”

```suggestion
当通过[继承 `Object`](/zh-CN/docs/Web/JavaScript/Reference/Classes/extends#扩展对象) 的类的构造函数中的 [`super()`](/zh-CN/docs/Web/JavaScript/Reference/Operators/super) 隐式调用 `Object()` 时，它以 `new.target.prototype` 为原型初始化一个新对象。传递给 `super()` 的任意值都将被忽略——例如，即使你传递一个数字，构造函数中的 `this` 值也不会变成 {{jsxref("Number")}} 实例。
```

**Review by yin1999 — APPROVED** [review]

**Comment by yin1999** [comment]
Thanks

File: files/zh-cn/web/javascript/reference/global_objects/object/object/index.md (hunk 1 above)
Comment (robert-jx) [reply files/zh-cn/web/javascript/reference/global_objects/object/object/index.md]: 了解😁

**Pull request merged**